This patch disables embedding of Windows manifests into the output files because
the Second Life build system embeds a different, custom manifest.

It is specific to the Second Life build and only ever be applied for builds 
that are to be used in the Second Life client.

---
diff -u a/mkspecs/win32-msvc2005/qmake.conf b/mkspecs/win32-msvc2005/qmake.conf
--- a/mkspecs/win32-msvc2005/qmake.conf
+++ b/mkspecs/win32-msvc2005/qmake.conf
@@ -6,7 +6,7 @@
 
 MAKEFILE_GENERATOR      = MSVC.NET
 TEMPLATE                = app
-CONFIG                  += qt warn_on release incremental flat link_prl precompile_header autogen_precompile_source copy_dir_files debug_and_release debug_and_release_target embed_manifest_dll embed_manifest_exe
+CONFIG                  += qt warn_on release incremental flat link_prl precompile_header autogen_precompile_source copy_dir_files debug_and_release debug_and_release_target embed_manifest_exe
 QT                      += core gui
 DEFINES                 += UNICODE WIN32 QT_LARGEFILE_SUPPORT
 QMAKE_COMPILER_DEFINES  += _MSC_VER=1400 WIN32
@@ -51,7 +51,7 @@
 QMAKE_RUN_CXX_IMP_BATCH = $(CXX) -c $(CXXFLAGS) $(INCPATH) -Fo$@ @<<
 
 QMAKE_LINK              = link
-QMAKE_LFLAGS            = /NOLOGO
+QMAKE_LFLAGS            = /MANIFEST:NO
 QMAKE_LFLAGS_RELEASE    = /INCREMENTAL:NO
 QMAKE_LFLAGS_DEBUG      = /DEBUG
 QMAKE_LFLAGS_CONSOLE    = /SUBSYSTEM:CONSOLE
